### PR TITLE
Prepare to publish AppCheck package

### PR DESCRIPTION
**Pull request by hsubox76**
Prepare AppCheck to be published in the next release.

- Make 3 packages public
  - app-check
  - app-check-types
  - app-check-interop-types
- Add app-check import to
  - firebase/index.ts
  - firebase/index.cdn.ts
- Add firebase/app-check folder with
  - package.json
  - index.ts
- firebase/package.json
  - Add dependency
  - Add to components list
- Add to toc.yaml (JS only)
- Set up tokens for canary publish
- Ensure local dependencies are up to date in app-check/package.json (redo after regular release tomorrow)

### CODE REVIEW

File: packages/app-check-interop-types/package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "@firebase/app-check-interop-types",
-  "private": true,
   "version": "0.1.0",
```
Comment (Feiyang1) [inline packages/app-check-interop-types/package.json:3]: change versions to `0.0.1` so minor bumps them to `0.1.0`

File: packages/firebase/package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "@firebase/performance": "0.4.11",
     "@firebase/remote-config": "0.1.36",
     "@firebase/analytics": "0.6.9",
+    "@firebase/app-check": "0.1.0",
```
Comment (Feiyang1) [inline packages/firebase/package.json:60]: update to `0.0.1`

File: packages/app-check-interop-types/package.json (hunk 1 above)
Comment (hsubox76) [reply packages/app-check-interop-types/package.json]: Done.

File: packages/firebase/package.json (hunk 2 above)
Comment (hsubox76) [reply packages/firebase/package.json]: Almost forgot this part, thanks, done.

**Review by Feiyang1 — APPROVED** [review]

**Pull request merged**